cmd/yggd/config.go

```diff
@@ -10,17 +10,18 @@ import (
 )
 
 const (
-	cliLogLevel      = "log-level"
-	cliCertFile      = "cert-file"
-	cliKeyFile       = "key-file"
-	cliCaRoot        = "ca-root"
-	cliServer        = "server"
-	cliSocketAddr    = "socket-addr"
-	cliClientID      = "client-id"
-	cliPathPrefix    = "path-prefix"
-	cliProtocol      = "protocol"
-	cliDataHost      = "data-host"
-	cliExcludeWorker = "exclude-worker"
+	cliLogLevel        = "log-level"
+	cliCertFile        = "cert-file"
+	cliKeyFile         = "key-file"
+	cliCaRoot          = "ca-root"
+	cliServer          = "server"
+	cliSocketAddr      = "socket-addr"
+	cliClientID        = "client-id"
+	cliPathPrefix      = "path-prefix"
+	cliProtocol        = "protocol"
+	cliDataHost        = "data-host"
+	cliExcludeWorker   = "exclude-worker"
+	cliWorkerConfigDir = "worker-config-dir"
```

Comment:
Go fmt will always surprise me. You add one line, and all the structure is reformatted, and part of the commit ... 🤣